SuperBuild/External_JsonCpp.cmake

```diff
@@ -0,0 +1,79 @@
+
+set( proj JsonCpp )
+
+# Set dependency list
+set(${proj}_DEPENDENCIES "")
+
+# Include dependent projects if any
+ExternalProject_Include_Dependencies(${proj} PROJECT_VAR proj DEPENDS_VAR ${proj}_DEPENDENCIES)
+
+# Sanity checks
+if(${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
+  message(FATAL_ERROR "Enabling ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj} is not supported !")
+endif()
+if(DEFINED ${proj}_DIR AND NOT EXISTS ${${proj}_DIR})
+  message(FATAL_ERROR "${proj}_DIR variable is defined but corresponds to nonexistent directory")
+endif()
+
+if(NOT DEFINED ${proj}_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
+
+  set(EXTERNAL_PROJECT_OPTIONAL_ARGS)
+
+  if(NOT DEFINED git_protocol)
+    set(git_protocol "git")
+  endif()
+
+  if (NOT WIN32)
+    list(APPEND EXTERNAL_PROJECT_OPTIONAL_ARGS
+      -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
+      )
+  endif()
+
+  ExternalProject_Add(${proj}
+    ${${proj}_EP_ARGS}
+    GIT_REPOSITORY "${git_protocol}://github.com/KitwareMedical/jsoncpp.git"
```

Comment:
> @jcfr I am not clear about your comment. open-source-parsers/jsoncpp#467 is integrated. 

Great. This was on oversight. In that case, we should have look at the other commits to also integrate them if relevant.

My other comment was some suggestion to alleviate the concern raised.

We are all set then.

The two other commits are quit minor.

@vovythevov Could you look into using the latest release of jsoncpp ?